semantic/src/site/collections/grid.overrides

```diff
@@ -1,3 +1,27 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.grid {
+  margin: 10px;
+  border-radius: 6px;
+}
+
+/* Table Striping */
+.stripes .ui.grid:nth-child(even) {
+  background: @surfacesLight;
+}
+
+.stripes .ui.grid:nth-child(even) .stripedlabel {
+  background: white;
+}
+
+.stripes .ui.grid:nth-child(odd) .stripedlabel {
+  background: @surfacesLight;
+}
+
+.ui[class*='middle aligned'].grid > .column:not(.row) {
+  padding: 0px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
```

Comment:
Hey, I don't think it's a good idea to go over-riding the default styles for all Grids, since Grids are already in use in other places. Primarily, the main application form page from my last PR (#647) now has weirdly wide margins due to these changes, which makes the Progress bar look manky.

How it should be:

<img width="1122" alt="Screen Shot 2021-05-02 at 11 23 10 PM" src="https://user-images.githubusercontent.com/5456533/116812051-b5507c80-aba0-11eb-92b0-616c5fcc342f.png">

How it is now:
<img width="1128" alt="Screen Shot 2021-05-02 at 11 24 13 PM" src="https://user-images.githubusercontent.com/5456533/116812055-baadc700-aba0-11eb-9247-eedc2f08625d.png">


Probably the best thing to do is to use `#id`s on the specific elements you want to over-ride. Since grids are very much a "custom" thing that changes on a case-by-case basis, we should probably leave the defaults as is (unless you think there is a strong reason not to?)

Reply:
Sorry, I didn't realise it had changed the layout of the In progress page : (
I thought that the `border-radius: 6px` would be something global to all grids, but didn't realise the margin was going to be a bad decision. I'll add that to a specific `id`. All the other styles are under a **className** `.stripes`, so should be ok, right?

Reply:
> All the other styles are under a **className** `.stripes`, so should be ok, right?

Yeah, looks like it.

Reply:
I tried something else, have changed the margin inside each row, and started using the Grid.Row with this purpose on `ReviewSectionRow` and `AssignmentSectionRow`, both are contained in a semantic component (Segment) that is using the `.stripes` class. Hope that is a better solution to this: 32c935a